pom.xml

```diff
@@ -138,6 +133,30 @@
       <version>2.30</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>matrix-auth</artifactId>
+      <version>1.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>matrix-project</artifactId>
+      <version>1.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency> <!-- required to workaround the missing CookieSpecProvider class -->
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+      <version>4.5.2</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpcore</artifactId>
+      <version>4.4.4</version>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
 
   <build>
```

Comment:
The build section can be removed completely.  `maven-release-plugin`, `findbugs-maven-plugin` and `maven-hpi-plugin` are already defined in the parent pom.